tree/tree/test/BulkApi.cxx

```diff
@@ -152,3 +220,38 @@ TEST_F(BulkApiTest, fastRead)
    printf("TTreeReaderFast: Total elapsed time (seconds) for bulk APIs: %.2f\n", sw.RealTime());
    delete hfile;
 }
+
+TEST_F(BulkApiTest, BulkInMem)
+{
+   TTree t("t","t");
+   int i = 3;
+   t.Branch("fAlpha",&i);
+   for(i = 0; i < 100000; ++i)
+      t.Fill();
+
+   TBufferFile buf(TBuffer::EMode::kWrite, 32*1024);
+   auto &r = t.GetBranch("fAlpha")->GetBulkRead();
+
+   {
+      auto s = r.GetBulkEntries(0, buf);
+      ASSERT_EQ(s, 7982) << "Did not read the expected number of entries.";
```

Comment:
GTest works better if the expected value comes first in the equality checks.  Otherwise the error message reads "upside down".

Reply:
fair enough.